__tests__/app.test.js

```diff
@@ -121,16 +118,52 @@ describe('/api/articles', () => {
         .get('/api/articles')
         .expect(200)
         .then(({ body }) => {
-            const {articles} = body;
-            const dates = [];
-            expect(articles.length > 0).toBe(true);
+            expect(body).toBeSortedBy('created_at', {
+                descending: true,
+            })
+        })
+    })
+})
 
-            articles.forEach((article) =>{
-                dates.push(article.created_at);
+describe('/api/articles/:id/comments', () => {
+    describe('STATUS 200', () => {
+        test('200: GET /api/articles/:id/comments - responds with an array of comment objects containing the relevant keys', () => {
+            return request(app)
+            .get('/api/articles/1/comments')
+            .expect(200)
+            .then(({ body }) => {
+                body.forEach((comment) => {
+                    expect(comment).toMatchObject({
+                        comment_id: expect.any(Number),
+                        votes: expect.any(Number),
+                        created_at: expect.any(String),
+                        author: expect.any(String),
+                        body: expect.any(String),
+                        article_id: expect.any(Number)
```

Comment:
We can be more specific with the article_id here as we know what article we have queried